examples/twittermap/web/app/controllers/TwitterMapApplication.scala

```diff
@@ -113,49 +107,23 @@ class TwitterMapApplication @Inject()(val wsClient: WSClient,
     override def receive = {
       case msg: JsValue =>
         val queryWords = (msg \\ "keyword").head.as[String]
-        val location = Array{(msg \\ "location").head.as[Array[Double]]}
-        val locs = Array(Array(location.head(1),location.head(0)),Array(location.head(3),location.head(2)))
-        val center = Array((locs(0)(0) + locs(1)(0))/2.00,(locs(0)(1)+locs(1)(1))/2.00)
-        val centerLoc = new GeoLocation(center(1),center(0))
-        val unit = Query.Unit.km
-        /*
-        Note: Twitter Search API does not support bounding box, the Geo information can only be specified
-        By using center location and radius.
-        Center location is calculated by the center point of bounding box(rectangle)
-        Radius is calculated by the difference of latitude between left top corner and right bottom corner.
-        Using latitude rather than longitude
-        since the ratio between latitude to KM is almost constant
-        But ratio between Longitude to KM varies largerly.
-        ------------------
-        |     * * *      |
-        |    *     *     |
-        |      * *       |
-        ------------------
-        */
-        val radius = Math.abs(locs(0)(1) - locs(1)(1)) * 111
-        var tweetArray = Json.arr()
-        val cb2 = new ConfigurationBuilder
-        cb2.setDebugEnabled(true)
-          .setOAuthConsumerKey(liveTweetConsumerKey)
-          .setOAuthConsumerSecret(liveTweetConsumerSecret)
-          .setOAuthAccessToken(liveTweetToken)
-          .setOAuthAccessTokenSecret(liveTweetTokenSecret)
-        val tf = new TwitterFactory(cb2.build)
-        val twitterAPI = tf.getInstance
-        val query = new twitter4j.Query
-        var desiredTweetAmount = (liveTweetQueryInterval / liveTweetUpdateRate).toInt
-        val resultType = twitter4j.Query.ResultType.recent
-        query.setQuery(queryWords)
-        query.setCount(desiredTweetAmount)
-        query.setResultType(resultType)
-        query.setGeoCode(centerLoc,radius,unit)
-        val tweetsResult = twitterAPI.search(query).getTweets
-        for(i <- 0 to tweetsResult.size()-1){
-          val status = tweetsResult.get(i)
-          if (status.isRetweet == false){
-            tweetArray = tweetArray :+ (Json.obj("id" -> status.getId.toString))
-          }
 
+        val url = "https://twitter.com/i/search/timeline?f=tweets&vertical=news&q="+queryWords+"&src=typd"
```

Comment:
Since this new way could be deprecated really easily in the future due to many reasons (e.g. Twitter updated the timeline page structure or limit the connection within OAth verification, etc. ), is it a better idea to keep the previous version as well by adding a flag in the configuration file to turn this hack version ON or OFF?
Just my thoughts, not sure which one is better. We may need the input from @chenlica , what do you think?